subsys/random/rand32_entropy_common.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <drivers/entropy.h>
+
+void sys_rand_access_grant(struct k_thread *thread)
```

Comment:
This function is small enough that I wonder if we could just declare it inline in the header.